libs/opengl/include/mrpt/opengl/CPointCloudColoured.h

```diff
@@ -299,6 +299,10 @@ class PointCloudAdapter<mrpt::opengl::CPointCloudColoured>
 	inline size_t size() const { return m_obj.size(); }
 	/** Set number of points (to uninitialized values) */
 	inline void resize(const size_t N) { m_obj.resize(N); }
+	/** Does nothing as of now */
+	inline void setDimensions(const size_t& height, const size_t& width)
```

Comment:
same here...

Reply:
ah! Please, add this change to the docs/doxygen-pages/changelog.h list of changes...

Reply:
Made the changes you requested!
And did a git commit --amend, to make it one commit. Hope that's okay.